frontend/components/vault/VaultLiquidationTransactionFlow.vue

```diff
@@ -0,0 +1,147 @@
+                :wallet-address="walletAddress"
+                :is-loading="isConnectingWallet"
+                :is-explanations-shown="isExplanationsShown"
+                :is-correct.sync="isWalletConnected"
+                @connectWallet="$emit('connectWallet')"
+                @disconnectWallet="$emit('disconnectWallet')"
+            />
+            <VaultLiquidationLimitsCheckPanel
+                :liquidation-limits="liquidationLimits"
+                :collateral-type="vaultTransaction.collateralType"
+                :debt-dai="vaultTransaction.debtDai"
+                :incentive-relative-dai="vaultTransaction.incentiveRelativeDai"
+                :incentive-constant-dai="vaultTransaction.incentiveConstantDai"
+                :is-refreshing="isRefreshingLimits"
+                :is-explanations-shown="isExplanationsShown"
+                :is-correct.sync="areLimitsNotReached"
+                @refreshLimits="$emit('refreshLimits')"
+            />
+            <VaultLiquidationPanel
+                :vault-id="vaultTransaction.id"
+                :network="vaultTransaction.network"
+                :vault-state="vaultTransaction.state"
+                :wallet-address="walletAddress"
+                :disabled="!isWalletConnected || !areLimitsNotReached"
+                :is-liquidating="isLiquidating"
+                :is-explanations-shown="isExplanationsShown"
+                @liquidate="$emit('liquidate', $event)"
+            />
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { Alert, Button } from 'ant-design-vue';
+import { LiquidationLimits, VaultTransactionNotLiquidated } from 'auctions-core/dist/src/types';
+import TransactionFeesTable from '../auction/TransactionFeesTable.vue';
+import { generateLink } from '../../helpers/generateLink';
+import TimeTill from '../common/formatters/TimeTill.vue';
+import FormatAddress from '../common/formatters/FormatAddress.vue';
+import VaultLiquidationLimitsCheckPanel from '../panels/VaultLiquidationLimitsCheckPanel.vue';
+import VaultLiquidationPanel from '../panels/VaultLiquidationPanel.vue';
+import VaultLiquidationTransactionTable from './VaultLiquidationTransactionTable.vue';
+import WalletConnectionCheckPanel from '~/components/panels/WalletConnectionCheckPanel.vue';
+import TextBlock from '~/components/common/other/TextBlock.vue';
+import Explain from '~/components/common/other/Explain.vue';
+
+export default Vue.extend({
+    components: {
+        VaultLiquidationPanel,
+        VaultLiquidationLimitsCheckPanel,
+        FormatAddress,
+        TimeTill,
+        TransactionFeesTable,
+        VaultLiquidationTransactionTable,
+        TextBlock,
+        Alert,
+        Button,
+        WalletConnectionCheckPanel,
+        Explain,
+    },
+    props: {
+        vaultTransaction: {
+            type: Object as Vue.PropType<VaultTransactionNotLiquidated>,
+            required: true,
+        },
```

Comment:
You accept `VaultTransactionNotLiquidated` type inside this panel, but inside the panel handles liquidated state. How is it possible?

Suggestion: keep the prop as is and hide the whole `VaultLiquidationTransactionFlow` in case the vault is liquidated, showing only the alert in the parent?

Reply:
I don't think this will be a big issue! I changed the type to `VaultTransaction` which has both sub types. I already correctly handled the finished auction state, just forgot to update the type!